frictionless/project/filesystem.py

```diff
@@ -60,6 +60,17 @@ def create_file(
         path = self.get_secure_relpath(path)
         return path
 
+    def create_file(self, path: str, *, folder: Optional[str] = None) -> str:
+        if folder:
+            folder = self.get_secure_fullpath(folder)
+            assert self.is_folder(folder)
+        resource = Resource(path)
+        name = str(path.split("/")[-1])
+        path = self.get_secure_fullpath(folder, name, deduplicate=True)
+        resource.write(path)
```

Comment:
Instead of `resource.write` it will be better to `filesystem.create_file(bytes=resource.read_bytes())` (see above as well)